content/10-humanities-examples.rst

```diff
@@ -0,0 +1,40 @@
+.. _example-humanities:
+
+Use cases: humanities
+==============================
+
+.. questions::
+
+   - How are GPUs used in practical humanities problems?
+   - Why are they necessary in fields outside the hard sciences?
+
+.. objectives::
+
+   - o1
+   - o2
+
+.. instructor-note::
+
+   - X min teaching
+   - X min exercises
+
+
+.. keypoints::
+
+GPU computing in the humanities
```

Comment:
I suggest adding the formatting to make paragraph titles more clear:

```suggestion
GPU computing in the humanities
-------------------------------
```

<details>
<summary>Result together with other suggestions (click to show)</summary>

![image](https://user-images.githubusercontent.com/933873/226117517-47ee998f-6322-4dfa-afa3-1783d9018cce.png)

</details>